arch/risc-v/include/limits.h

```diff
@@ -55,37 +55,44 @@
 /* These change on 32-bit and 64-bit platforms */
 
 #if defined(CONFIG_ARCH_RV32)
+#  define LONG_MIN  (-LONG_MAX - 1)
+#  define LONG_MAX  2147483647L
+#  define ULONG_MAX 4294967295UL
 
-#define LONG_MIN    (-LONG_MAX - 1)
-#define LONG_MAX    2147483647L
-#define ULONG_MAX   4294967295UL
-
-#define LLONG_MIN   (-LLONG_MAX - 1)
-#define LLONG_MAX   9223372036854775807LL
-#define ULLONG_MAX  18446744073709551615ULL
+#  define LLONG_MIN  (-LLONG_MAX - 1)
+#  define LLONG_MAX  9223372036854775807LL
+#  define ULLONG_MAX 18446744073709551615ULL
 
 /* A pointer is 4 bytes */
 
-#define PTR_MIN     (-PTR_MAX - 1)
-#define PTR_MAX     2147483647
-#define UPTR_MAX    4294967295U
-
+#  define PTR_MIN   (-PTR_MAX - 1)
+#  define PTR_MAX   2147483647
+#  define UPTR_MAX  4294967295U
 #endif /* defined(CONFIG_ARCH_RV32) */
 
 #if defined(CONFIG_ARCH_RV64)
```

Comment:
Not a big deal, but possibly we can re-arrange this to follow style like in
```
#ifdef CONFIG_ARCH_RV64
typedef signed int         _int32_t;
typedef unsigned int       _uint32_t;

typedef signed long        _int64_t;
typedef unsigned long      _uint64_t;
#else /* CONFIG_ARCH_RV64 */
typedef signed long        _int32_t;
typedef unsigned long      _uint32_t;

typedef signed long long   _int64_t;
typedef unsigned long long _uint64_t;
#endif /* CONFIG_ARCH_RV64 */
```
instead of two separate `#if` cases.
Please feedback if you will do it or not

Reply:
Done.